commands.py

```diff
@@ -141,7 +168,66 @@ def run(self, events, *args, **kwargs):
     else:
       body = ERROR_NOT_AUTHORIZED_TO_DELETE
 
-    return RSVPCommandResponse(body, events, 'stream')
+    return RSVPCommandResponse(events, RSVPMessage('stream', body))
+
+
+class RSVPMoveCommand(RSVPEventNeededCommand):
+  regex = r'move (?P<destination>.+)$'
+
+  def run(self, events, *args, **kwargs):
+    event_id = kwargs.pop('event_id')
+    sender_id = kwargs.pop('sender_id')
+    event = kwargs.pop('event')
+    destination = kwargs.pop('destination')
+
+    # Check if the issuer of this command is the event's original creator.
+    # Only he can delete the event.
+    creator = event['creator']
+
+    # check and make sure a valid Zulip stream/topic URL is passed
+    if not destination: 
+      body = ERROR_MISSING_MOVE_DESTINATION
+
+    elif creator != sender_id:
+      body = ERROR_NOT_AUTHORIZED_TO_DELETE
+
+    else:
+      # split URL into components
+
+      stream, topic = util.narrow_url_to_stream_topic(destination)
+
+      if stream is None or topic is None:
+        body = ERROR_BAD_MOVE_DESTINATION % destination
+      else:
+        new_event_id = stream + "/" + topic
+        body = MSG_EVENT_MOVED % (new_event_id, destination)
+
+        old_event = events.pop(event_id)
+
+        # need to make sure that there's no duplicate here!
+        # also, ideally we'd make sure the stream/topic existed & create it if not.
+        # AND send an 'init' notification to that new stream/toipic. Hm. what's the 
+        # best way to do that? Allow for a parameterized init? It's always a reply, not a push. 
+        # Can we return MULTIPLE messages instead of just one?
+
+        events.update(
+          { 
+            new_event_id: {
+              'name': topic,
+              'description': old_event['description'],
+              'place': old_event['place'],
+              'creator': old_event['creator'],
+              'yes': old_event['yes'],
+              'no': old_event['no'],
+              'maybe': old_event['maybe'],
+              'time': old_event['time'],
+              'limit': old_event['limit'],
+              'date': old_event['date'],
+            }
+          }
+        )
+
+    return RSVPCommandResponse(events, RSVPMessage('stream', body), RSVPMessage('stream', "I'm an RSVP event now", stream, topic))
```

Comment:
Let's use more formal speech for the message, something like "This thread is now an RSVPBot event! Type `rsvp help` for more options".

Also, the message needs to live as a constant with all the other messages :8ball: